_posts/2024-08-12-AI-chatting.md

```diff
@@ -0,0 +1,93 @@
+
+## Which AI provider should I use?
+
+We recomend you chosing the OpenAI.
+
+For Mistral AI you need to make a subscription, while for OpenAI you can send money one time.
+
+Hugging Face gives you access to numerous count of models for free. But it will take a very long time for Hugging Face to find a free computer resources for you, and the response time will be also long.
+
+## How to get an API key and what should I do after getting one?
+
+You can refer to [AI user documentation](https://docs.jabref.org/ai/ai-providers-and-api-keys).
+
+## AI preferences
+
+Here are some new options in the JabRef preferences.
+
+![AI preferences](../img/AiPreferences.png)
+
+- "Enable AI functionality in JabRef": by default it's turned off, so you need to check this option, if you want to use the new AI features
+- "AI provider": you can choose either OpenAI, Mistral AI, or Hugging Face
+- "Chat model": choose the model you like (for OpenAI we recommend `gpt-4o-mini`, as it the cheapest and fastest)
+- "API token": here you write your API token
+- "Expert settings": here you can change the parameters that affect how AI will generate your answers. If you don't understand the meaning of those settings, don't worry! We have experimented a lot and found the best parameters for you! But if you are curious, then you can refer to [user documentation]()
+
+## BONUS
+
+Have you ever wanted to try running a local LLM model on your computer with 100% privacy saved? We got you.
+
+We wrote a tutorial of how to install and use `ollama` and integrate it in JabRef. You will find more information at the [user documentation](https://docs.jabref.org/ai/local-llm).
+
+## End
+
+Thank you for using JabRef and checking out the AI functionality!
+
+This is only the first step of the Google Summer of Code project, so stay tuned for new AI features!
+
+Currently the summarization chatting is under development, so some bugs may occur.
+
+We value your opinion and we want to know: What AI features would you like to see in JabRef in future? What LLM providers would you like to have in next versions?
+
+(List of LLM providers: [Huggingface](https://huggingface.co/pricing#endpoints), [Azure](https://azure.microsoft.com/en-us/products/#ai-machine-learning), [Together.AI](https://www.together.ai/), [Google](https://ai.google/discover/our-models/), [MistralAI](https://mistral.ai/), [Anthrophic](https://www.anthropic.com/), [Meta](https://llama.meta.com/), ... )
```

Comment:
Move this up to "Which AI provider should I use?" - Also explain "..." (maybe, just leave out the dots - or say that OpenAI-API compatible providers can be used.

Reply:
I've changed the text to: "What LLMs or other integrations would you like to have in next versions?". I think this is more known for users rather than AI or LLM provider.

I removed the text. Will that be okay?